app/src/services/converter.js

```diff
@@ -0,0 +1,69 @@
+const logger = require('logger');
+const ogr2ogr = require('ogr2ogr').default;
+const XLSX = require('xlsx');
+const fs = require('fs');
+const path = require('path');
+const unzipper = require('unzipper');
+
+class OGRConverter {
+
+    static async convert(ctx) {
+        logger.info('Converting file...', ctx.request.body);
+        logger.debug(`[OGRRouterV2 - convertV2] request: ${JSON.stringify(ctx.request)}`);
+        ctx.assert(ctx.request.files && ctx.request.files.file, 400, 'File required');
+        logger.debug(`[OGRRouterV2 - convertV2] file data: ${JSON.stringify(ctx.request.files.file)}`);
+
+        let result;
+        logger.info(`[OGRRouterV2 - convertV2] file type: ${ctx.request.files.file.type}`);
+        logger.debug(`[OGRRouterV2 - convertV2] file size: ${ctx.request.files.file.size}`);
+
+        if (ctx.request.files.file.mimetype === 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet') {
+            logger.info('[OGRRouterV2 - convertV2] It is an excel file');
```

Comment:
please modify this and all logger lines, since this is not the OGRRouterV2, and it is not the convertV2 method anymore.